lineman/app/components/thread_page/thread_page_controller.coffee

```diff
@@ -9,6 +9,8 @@ angular.module('loomioApp').controller 'ThreadPageController', ($routeParams, $l
     @group = @discussion.group()
     $rootScope.$broadcast('viewingThread', @discussion)
     MessageChannelService.subscribeTo("/discussion-#{@discussion.key}", @onMessageReceived)
+  , (error) ->
```

Comment:
We only broadcast the 'pageError' event when the initial 'Get me the thread / group' request fails. We'll have to broadcast other types of events for other types of failures, I'd imagine.